articles/connect-end-user-to-wifi-with-certificate.md

```diff
@@ -52,7 +53,16 @@ To connect end users to W-Fi or VPN with DigiCert certificates, we'll do the fol
 
 When Fleet delivers the profile to your hosts, Fleet will replace the variables. If something goes wrong, errors will appear on each host's **Host details > OS settings**.
 
-If you resend the profile (select **Resend** in **Host details > OS settings**), Fleet will get a new certificate and create a new seat in DigiCert, which will take 1 license. If you want to revoke a license, in DigiCert, head to [**Trust Lifcycle Manager > Account > Seats**](https://demo.one.digicert.com/mpki/account/seats) and remove the seat.
+Fleet will automatically resend the profile to renew the certificate 30 days before expiration.
+
+If end user is on vacation (offline more than 30 days) and certificate expires, they should connect to a different network so new certificate is delivered and then they can connect back to a corporate Wi-Fi and VPN.
+
+DigiCert device type seats can have multiple certificates assigned, and the user will pay for only one license. Only duplicate certificates (those with the same CN and seat ID) can be assigned to a one device type seat.
```

Comment:
```suggestion
Each DigiCert device type seat (license) can have multiple certificates only if they have the same CN and seat ID. If a new certificate has a different CN, a new DigiCert license is required.
```

Tried to make this understandable by someone who knows nothing about DigiCert. Accurate?